src/main/java/com/example/WorldOfAnimals/models/AnimalBreed.java

```diff
@@ -0,0 +1,29 @@
+package com.example.WorldOfAnimals.models;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table
+public class AnimalBreed {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column
```

Comment:
Where colomn name and options ? 

@Column(name="name", nullable=false, unique=true)

Reply:
done